Added about.md for image-to-text task (#686)
huggingface/hub-docs: tasks/src/image-to-text/about.md

## Use Cases

> 

Reviewer: ```suggestion
```

### Image Captioning

> 

Reviewer: ```suggestion
```

Image Captioning is the process of generating textual description of an image. 

> 

Reviewer: ```suggestion
```

> This can help the visually impaired to understand what's happening in their surroundings.

Reviewer: ```suggestion
This can help the visually impaired people to understand what's happening in their surroundings.
```

>  - For more details on this task refer [here](https://huggingface.co/docs/transformers/main/en/tasks/image_captioning)

Reviewer: ```suggestion
```
This already exists in useful resources so I'd say we remove it

 

### Optical Character Recognition (OCR)

> 

Reviewer: ```suggestion
```

> OCR converts the text present in an image(Ex: Scanned document) to machine readable formats.

Reviewer: ```suggestion
OCR models convert the text present in an image, e.g. a scanned document, to text.
```

Try it on [🤗 Spaces](https://huggingface.co/microsoft/trocr-base-handwritten)

> ## Task Variants

Reviewer: I think this is not needed as of now 😊

You can contribute variants of this task [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/image-to-text/about.md).

## Inference

> 

Reviewer: ```suggestion
```

> ### Image Captioning

Reviewer: Can you write a small explanation on what below snippet is for? Same with OCR.

```python
from transformers import pipeline

captioner = pipeline(model="ydshieh/vit-gpt2-coco-en")
captioner("https://huggingface.co/datasets/Narsil/image_dummy/raw/main/parrots.png")
## [{'generated_text': 'two birds are standing next to each other '}]
```

> ### OCR

Reviewer: Can below script be simplified? pinging @NielsRogge in case he has suggestions.
